recipe/meta.yaml

```diff
@@ -61,4 +62,4 @@ extra:
     - dotsdl
```

Comment:
We should also add @jthorton

Reply:
Oh and @ianmkenney

Reply:
let's do this separately so we don't miss anyone - including Hannah